.github/workflows/fetch-new-data.yaml

```diff
@@ -0,0 +1,57 @@
+name: Update Open-Data
+on:
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
Having this run everyday is probably too much IMO (Weekly could be more suitable). Also, I would prefer if you change the time from 8AM to late night/very early morning.

Reply:
Changed to sunday midnight.